e2e/log.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
if this is a copy from some other place, please mention the source in the commit message.

Reply:
done PTAL